tests/test_glue/helpers.py

```diff
@@ -66,8 +66,8 @@ def get_table(client, database_name, table_name):
     return client.get_table(DatabaseName=database_name, Name=table_name)
 
 
-def get_tables(client, database_name):
-    return client.get_tables(DatabaseName=database_name)
+def get_tables(client, database_name, expression=""):
+    return client.get_tables(DatabaseName=database_name, Expression=expression)

```

Comment:
```suggestion
def get_tables(client, database_name, expression=None):
    if expression:
        return client.get_tables(DatabaseName=database_name, Expression=expression)
    else:
        return client.get_tables(DatabaseName=database_name)
```
Most of the tests do not provide an expression - and we should verify that it still works, even if an expression is not provided.

You may want to test this out locally, I expect some tests to fail with this change